met/docs/Users_Guide/release-notes.rst

```diff
@@ -5,7 +5,34 @@ When applicable, release notes are followed by the GitHub issue number which
 describes the bugfix, enhancement, or new feature:
 `MET GitHub issues. <https://github.com/dtcenter/MET/issues>`_
 
-MET Version 10.1.0-beta1 release notes (|release_date|)
+MET Version 10.1.0-beta2 release notes (|release_date|)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Fix the findings from SonarQube (`#1855 <https://github.com/dtcenter/MET/issues/1855>`_).
+* Fix gen_vx_mask so that the -input_field and -mask_field options are processed independently (`#1891 <https://github.com/dtcenter/MET/issues/1891>`_).
+* Enhance Stat-Analysis to write the GO Index and CBS Index into a new STAT line type (`#1788 <https://github.com/dtcenter/MET/issues/1788>`_).
+* Enhance MET to compute the CBS score (`#1031 <https://github.com/dtcenter/MET/issues/1031>`_).
+* Fix integer overflow in Grid-Diag (`#1886 <https://github.com/dtcenter/MET/issues/1886>`_).
+* Fix failure of Ensemble-Stat when verifying against gridded ECMWF GRIB1 files (`#1879 <https://github.com/dtcenter/MET/issues/1879>`_).
+* Fix consumption of too much memory by Stat-Analysis (`#1875 <https://github.com/dtcenter/MET/issues/1875>`_).
+* Fix failure in plot_tcmpr.R script when a directory is passed in with -lookin (`#1872 <https://github.com/dtcenter/MET/issues/1872>`_).
+* Enable TC-Pairs to only write output for a configurable list of valid times (`#1870 <https://github.com/dtcenter/MET/issues/1870>`_).
+* Add check for the start offset and data count are valid before calling NetCDF API (`#1852 <https://github.com/dtcenter/MET/issues/1852>`_).
+* Add new column, Scatter Index, for CNT line type (`#1843 <https://github.com/dtcenter/MET/issues/1843>`_).
+* Enable point2grid to support double type latitude/longitude variables(`#1838 <https://github.com/dtcenter/MET/issues/1838>`_).
+* Treat warnings from the documentation as errors to facilitate continuous integration with GHA (`#1819 <https://github.com/dtcenter/MET/issues/1819>`_).
+* Fix the output of point2grid which is flipped and rotated with lat/lon to lat/lon conversion (`#1817 <https://github.com/dtcenter/MET/issues/1817>`_).
+* Update versions of software on Cheyenne and Casper for beta 2 release installation (`#1807 <https://github.com/dtcenter/MET/issues/1807>`_).
```

Comment:
```suggestion
```
While this issue is attached to the beta2 project board, it results in no changes to the actual repository. So we don't need to alert users about this.

Reply:
Great!  Thank you!  I was wondering about that, so I'm glad you're looking at this.  It felt strange to add it.  I guess I should have created a project board note instead of an issue...